tests/memoryview/memoryview.pyx

```diff
@@ -1090,19 +1090,19 @@ def optimised_index_of_slice(int[:,:,:] arr, int x, int y, int z):
 
 def test_assign_from_byteslike(byteslike):
     """
-    >>> test_assign_from_byteslike(bytes(b'hello'))
-    b'hello'
-    >>> test_assign_from_byteslike(bytearray(b'howdy'))
-    b'howdy'
+    >>> test_assign_from_byteslike(bytes(b'hello')) == b'hello'
+    True
```

Comment:
That's a bit less informative than getting the actual output, though. If the test fails, it will be unclear why. You can decode the resulting bytes string before returning it, and then just print it. That will show the same output in both Py2/3.

Reply:
I hate messing `bytes` and `str` just for comparability with Python 2: this test has nothing to do with string processing.
But this should be a fine compromise:
````py
>>> test_assign_from_byteslike(bytes(b'hello')).decode()
'hello'
````

Reply:
Except that in Py2:
```py
>>> b'123'.decode()
u'123'
```
Thus my suggestion to `print()` it.